docs/machine-learning/index.yml

```
@@ -1,69 +1,107 @@
+            url: resources/tasks.md
+          - text: How to choose an algorithm
+            url: how-to-choose-an-ml-net-algorithm.md
+          - text: Data transformations
+            url: resources/transforms.md
+
+   # Model Builder Card
+  - title: Use Model Builder 
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: What is Model Builder?
+            url: automate-training-with-model-builder.md
+      - linkListType: quickstart
+        links:
+          - text: Install Model Builder
+            url: how-to-guides/install-model-builder.md
+      - linkListType: tutorial
+        links:
+          - text: Predict prices - Model Builder
+            url: tutorials/predict-prices-with-model-builder.md
+          - text: Build a Razor Pages web app to analyze sentiment
+            url: tutorials/sentiment-analysis-model-builder.md
+ 
+  # API Card
+  - title: Get started with the API
+    linkLists:
+      - linkListType: tutorial
+        links:
+          - text: Build a TensorFlow image classifier with transfer learning
+            url: tutorials/image-classification.md
+          - text: Analyze website comment sentiment
+            url: tutorials/sentiment-analysis.md
+          - text: Detect objects in images using ONNX
+            url: tutorials/object-detection-onnx.md
+      - linkListType: how-to-guide
+        links:
+          - text: Load data from various sources  
+            url: how-to-guides/load-data-ml-net.md
+          - text: Prepare data for building a model
+            url: how-to-guides/prepare-data-ml-net.md
+          - text: Train & evaluate a model
+            url: how-to-guides/train-machine-learning-model-ml-net.md
+          - text: Make predictions with a trained model
+            url: how-to-guides/machine-learning-model-predictions-ml-net.md
+          - text: Save & load a trained model
+            url: how-to-guides/save-load-machine-learning-models-ml-net.md                         
+  
+  # CLI card
+  - title:  Use the ML.NET CLI
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Automate model training with the ML.NET CLI
+            url: automate-training-with-cli.md
+      - linkListType: quickstart
+        links:
+          - text: Install ML.NET CLI
+            url: how-to-guides/install-ml-net-cli.md
+      - linkListType: tutorial
+        links:
+          - text: Analyze sentiment - ML.NET CLI & AutoML
+            url: tutorials/mlnet-cli.md
+
+  # Reference card
+  - title: API reference
+    linkLists:
+      - linkListType: reference
+        links:
+          - text: ML.NET API reference  
+            url: https://docs.microsoft.com/dotnet/api/?view=ml-dotnet
+          - text: ML.NET API reference (preview)
+            url: https://docs.microsoft.com/dotnet/api/?view=ml-dotnet-preview
+          - text: ML.NET CLI reference
+            url: reference/ml-net-cli-reference
```

This link is broken. Should it be the following?

```suggestion
            url: reference/ml-net-cli-reference.md
```

---

@mairaw: Good catch! I'll fix it in the next commit.